README.md

```diff
@@ -47,6 +47,29 @@ bundle({
 });
 ```
 
+### Build for Synthetics Canary
+
+When deploying a lambda as Synthetics Canary, it requires the packaged js file
+saved under `nodejs/node_modules` within the packaged zip file. Please use the
+option `artifactPrefix` to specify the directory structure within the packaged
+zip file:
+
+```bash
+yarn blamda --entries src/my-service.ts --outdir build-output \
+--artifactPrefix "nodejs/node_modules" --node 14
```

Comment:
As-is, this PR adds support for the property in the NodeJS API, but not in the CLI. You'll need to update `cli.ts` and add this new option around line 29 if you want to expose this via the CLI.

Also: could we'd use the hyphenated pattern for the CLI argument (i.e. `artifact-prefix`), for consistency with the existing `include-aws-sdk` option?